src/pages/MainPage.jsx

```diff
@@ -39,13 +39,14 @@ export default function MainPage() {
             ? movies.map((movie) => <MovieCard movie={movie} key={movie.imdbID} />)
             : null}
           {currentTab === Tabs.Favourites && favouriteMovies?.length
-            ? favouriteMovies.map((movie) => <MovieCard movie={movie} key={movie.imdbID} />)
+            ? recommendedMovies.map((movie) => <MovieCard movie={movie} key={movie.imdbID} />)
```

Comment:
Are we not going to add a task for this? Should we let the students figure it out themselves? Would be interesting lol.

Reply:
Yeah so this one we're gonna see if they can spot the bug - we wanted them to be able to raise some defects of their own